tasks/discover-pdrs/schemas/config.json

```diff
@@ -37,6 +37,11 @@
       "default": false,
       "type": "boolean"
     },
+    "httpRequestTimeout": {
+      "description": "Timeout (in seconds) for retreival by HTTP/HTTPS provider",
+      "default": 300,
```

Comment:
I would probably steer away from specifying a default here, and the same for `discover-granules`. If a default is set here, schema validation will actually update the config to use those default values. That is going to make it confusing about which default is being used, the one here, or the one specified in the provider client as `HTTP_REQUEST_TIMEOUT_SECONDS`. Letting this just be `undefined` will then default to using that one default that's specified in the provider client.

Reply:
Hmm.    I was thinking this would help define to users seeking to understand the inputs what the default is as the only documentation of these parameters *is* the schema.    I take your point, but removing it likely means we need to maintain documentation, which feels bad.     Furthermore it makes the *component* have a default which insulates it a bit from provider changes. 

I hear the concern about complexity there.    I'm wondering if the better solution is to remove an explicit override of the crawler's built in default instead?

Reply:
See comment on line 39 of packages/ingest/HttpProviderClient.js